app/styles/_color.scss

```diff
@@ -96,6 +99,9 @@ body {
   --side-menu-active-background-dark: #303339;
   --side-menu-active-background-light: rgba(254, 77, 63, 0.1);
 
+  // special case variable for file-compare
+  --file-compare-dark-bg: #fbfbfb;
```

Comment:
we can use `--background-light`

Reply:
This is not used anymore. Will be removed.